packages/contracts/test/scripts/MigrateMetapool.t.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import "src/dollar/core/UbiquityDollarManager.sol";
+import "src/dollar/Staking.sol";
+import "src/dollar/mocks/MockBondingShareV2.sol";
+import "src/dollar/interfaces/IMetaPool.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import "forge-std/Test.sol";
+
+contract MetapoolMigrate is Test {
+    address admin = 0xefC0e701A824943b469a694aC564Aa1efF7Ab7dd;
+    UbiquityDollarManager manager =
+        UbiquityDollarManager(0x4DA97a8b831C345dBe6d16FF7432DF2b7b776d98);
+    Staking staking =
+        Staking(payable(0xC251eCD9f1bD5230823F9A0F99a44A87Ddd4CA38));
+    BondingShareV2 bonds =
+        BondingShareV2(0x2dA07859613C14F6f05c97eFE37B9B4F212b5eF5);
+    IERC20 uAD = IERC20(0x0F644658510c95CB46955e55D7BA9DDa9E9fBEc6);
+    IERC20 USD3CRV = IERC20(0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490);
+    IMetaPool v2Metapool =
+        IMetaPool(0x20955CB69Ae1515962177D164dfC9522feef567E);
+    IMetaPool v3Metapool =
+        IMetaPool(0x9558b18f021FC3cBa1c9B777603829A42244818b);
+
+    uint256 mainnet;
+    uint256 snapshot;
+    uint256 metaBalanceV3;
+    uint256 metaBalance;
+
+    function setUp() public {
+        mainnet = vm.createSelectFork("https://eth.ubq.fi/v1/mainnet");
```

Comment:
Seems that the fork selection is mixed. I think this is a reasonable default but in order to stay consitent with the other scripts, shouldn't it be loaded in from the `.env` and fall back to this if nothing?

Reply:
```function setUp() public {
        
        string memory forkURL = vm.envString("RPC_URL");

        if(keccak256(abi.encodePacked(forkURL)) != keccak256(abi.encodePacked(""))){
            mainnet = vm.createSelectFork(forkURL);
        } else {
            mainnet = vm.createSelectFork("https://eth.ubq.fi/v1/mainnet");
        }
        snapshot = vm.snapshot();
    }```

Reply:
This fails in CI cause of the lack of `.env`
I preserved the pattern above by commenting it out but have reverted to just using 
```mainnet = vm.createSelectFork("https://eth.ubq.fi/v1/mainnet");```

It'd be a seperate issue but refactoring our tests to use `vm.createSelectFork()` instead of passing it as a flag in the CLI might be worthwhile.